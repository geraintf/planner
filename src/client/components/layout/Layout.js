import React from 'react';

const Layout = (props) => {
    return (
        <div>
            { props.children }
        </div>
    )
};

Layout.propTypes = {};

export default Layout;