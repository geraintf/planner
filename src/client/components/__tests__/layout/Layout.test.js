import React from 'react';
import { shallow } from 'enzyme';

import Layout from '../../layout/Layout';

const mockChild = <span>Some Child Content</span>;

describe('Layout', () => {
  let output;

  beforeEach(() => {
    output = shallow(<Layout>{ mockChild }</Layout>);
  });

  it('should render correctly', () => {
    expect(output).toMatchSnapshot();
  });
});

