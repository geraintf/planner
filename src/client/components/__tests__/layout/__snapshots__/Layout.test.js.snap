// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render correctly 1`] = `
<div
  className="default-layout"
>
  <span>
    Some Child Content
  </span>
</div>
`;
